src/main/java/tutorpro/logic/commands/AddCommand.java

```diff
@@ -28,15 +28,17 @@ public class AddCommand extends Command {
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
             + PREFIX_ADDRESS + "ADDRESS "
-            + "[" + PREFIX_TAG + "TAG]...\n"
+            + PREFIX_LEVEL + "LEVEL "
+            + PREFIX_SUBJECT + "SUBJECT-grade ..."
+            + PREFIX_TAG + "TAG...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "98765432 "
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
             + PREFIX_LEVEL + "P5 "
-            + PREFIX_SUBJECT + "Math "
-            + PREFIX_SUBJECT + "English "
+            + PREFIX_SUBJECT + "Math-B "
```

Comment:
I really like the fact that you added the Subject identifier in, it really enhances the developer experience.